src/tools/tidy/src/features.rs

```diff
@@ -89,6 +89,15 @@ pub fn check(
             &src_path.join("test/ui"),
             &src_path.join("test/ui-fulldeps"),
             &src_path.join("test/compile-fail"),
+            // This allows `tidy` to pick up features that are `rustdoc`-specific.
+            // They won't be run by `tidy` but they will be run later in the
```

Comment:
I'm confused what it means to be 'run by `tidy`'. Is there something different between rustdoc-ui and the other tests?

Reply:
I was tired when I wrote that. I'll fix it in the next commit